packages/@okta/vuepress-site/.vuepress/redirects.json

```diff
@@ -245,7 +245,11 @@
   },
   {
     "path": "/docs/getting_started/tokens.html",
-    "redirect": "/docs/api/resources/tokens/index.html"
+    "redirect": "/authentication-guide/tokens/"
```

Comment:
@bretterer - When should this file be updated vs conductor.yml?

Reply:
the redirects.json is not effective anymore.  It would only work if we began using the 404 page provided by VuePress, or had a page existing in the repo that we wanted to redirect from, but we should just be using conductor.yml for every redirect